src/analysis/openstreetmap/qgsosmdatabase.h

```diff
@@ -82,7 +82,6 @@ class ANALYSIS_EXPORT QgsOSMDatabase
 
     QgsOSMTags tags( bool way, QgsOSMId id ) const;
 
-    //! \note available in Python bindings
```

Comment:
do you know why it was not available? 
because of the issue you get with QgsOSMTagCountPair or was there another reason?